pkg/pool-linear/contracts/euler/interfaces/IEulerTokenMinimal.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.7.0;
+
+interface IEulerTokenMinimal {
+    function decimals() external view returns (uint256);
+
+    function convertBalanceToUnderlying(uint256) external view returns (uint256);
```

Comment:
It'd be nice to have a link to some documentation/github repo of this token.

Reply:
I updated the interface docs.